src/syngen/ml/config/configurations.py

```diff
@@ -43,20 +43,21 @@ class InferConfig:
     metadata_path: Optional[str] = None
     random_seed: Optional[int] = None
     print_report: bool = False
+    both_keys: bool
 
     def set_paths(self):
         """
         Create paths which used in infer process
         """
         return {
-            "original_data_path": f"model_artifacts/tmp_store/{self.table_name}/input_data.csv",
-            "synthetic_data_path": f"model_artifacts/tmp_store/{self.table_name}/merged_infer.csv",
-            "draws_path": f"model_artifacts/tmp_store/{self.table_name}/draws",
-            "input_data_path": f"model_artifacts/tmp_store/{self.table_name}/input_data.csv",
-            "path_to_merged_infer": f"model_artifacts/tmp_store/{self.table_name}/merged_infer.csv",
-            "state_path": f"model_artifacts/resources/{self.table_name}/vae/checkpoints",
-            "tmp_store_path": f"model_artifacts/tmp_store/{self.table_name}",
-            "vae_resources_path": f"model_artifacts/resources/{self.table_name}/vae/checkpoints/",
-            "dataset_pickle_path": f"model_artifacts/resources/{self.table_name}/vae/checkpoints/model_dataset.pkl",
-            "fk_kde_path": f"model_artifacts/resources/{self.table_name}/vae/checkpoints/fk_kde.pkl"
+            "original_data_path": f"model_artifacts/tmp_store/{self.table_name[:-3] if self.both_keys else self.table_name}/input_data.csv",
```

Comment:
I think it would be better to create a new variable and check condition once instead of multiple times